src/app/modules/shared_models/message_model.nim

```diff
@@ -326,13 +358,12 @@ QtObject:
       self.updateItemAtIndex(1)
     self.countChanged()
 
-  proc replyDeleted*(self: Model, messageIndex: int) {.signal.}
-
   proc updateMessagesWithResponseTo(self: Model, messageId: string) =
     for i in 0 ..< self.items.len:
       if(self.items[i].responseToMessageWithId == messageId):
-        let ind = self.createIndex(i, 0, nil)
-        self.replyDeleted(i)
+        self.items[i].responseToMessage = ""
+        let index = self.createIndex(i, 0, nil)
+        self.dataChanged(index, index, @[ModelRole.ResponseToMessage.int])
```

Comment:
FYI, this will conflict with the work I did to fix the reply messages being empty sometimes.
My version also uses the data directly instead of calling some JSON function, so I also have the performance improvements.
Letting you know in case my PR is merged first. https://github.com/status-im/status-desktop/pull/8886

Reply:
I'll wait for your PR to be merged and will update this one.